libraries/AP_HAL_ChibiOS/hwdef/scripts/dma_resolver.py

```diff
@@ -169,7 +169,7 @@ def generate_DMAMUX_map_mask(peripheral_list, channel_mask, noshare_list, dma_ex
         base = idx % 16
         for i in range(16):
             found = None
-            for ii in list(range(base,16)) + list(range(0,base)):
```

Comment:
0,base is a drive-by?

Reply:
Please read the commit message when reviewing pull requests.
> libraries/AP_HAL_ChibiOS/hwdef/scripts/dma_resolver.py:172:57: PIE808 [*] Unnecessary `start` argument in `range`